edb/pgsql/metaschema.py

```diff
@@ -6378,10 +6364,16 @@ def _generate_sql_information_schema(
             pi.indisclustered,
             pi.indisvalid,
             pi.indcheckxmin,
-            pi.indisready,
+            CASE
+                WHEN COALESCE(is_id.t, FALSE) THEN TRUE
+                ELSE FALSE -- override so pg_dump won't try to recreate them
+            END AS indisready,
```

Comment:
This is equivalent to `COALESCE(is_id.t, FALSE)`, right? The `CASE` seems unnecessary.

Reply:
The CASE is detecting whether this is a primary key and has this form for a few different columns in the view.